core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalConfiguration.java

```diff
@@ -53,6 +54,29 @@ public static RaftJournalConfiguration defaults(ServiceType serviceType) {
         .setPath(new File(JournalUtils.getJournalLocation().getPath()));
   }
 
+  /**
+   * @param clusterAddresses addresses of all nodes in the Raft cluster
+   * @return true if clusterAddress have an object of same ip with local Address, otherwise false
+   */
+  public boolean checkContainIp(List<InetSocketAddress> clusterAddresses) {
+    boolean contain = false;
+    String clusterItemIp;
+    String localAddressIp =  NetworkAddressUtils.getLocalIpAddress((int) ServerConfiguration
+        .global().getMs(PropertyKey.NETWORK_HOST_RESOLUTION_TIMEOUT_MS));
+    try {
+      for (InetSocketAddress addr : clusterAddresses) {
+        clusterItemIp = InetAddress.getByName(addr.getHostName()).getHostAddress();
+        if (clusterItemIp.equals(localAddressIp)) {
+          contain = true;
+          break;
+        }
+      }
+    } catch (Throwable t) {
+      return contain;
```

Comment:
We need an error level log here for helping us to find the root cause why it return false.